.github/workflows/migration-tests.yml

```diff
@@ -164,6 +164,10 @@ jobs:
         with:
           fetch-depth: 1
 
+      - name: Modify path for libpq
+        if: matrix.os == 'macos-latest'
+        run: echo "/opt/homebrew/opt/libpq/bin" >> $GITHUB_PATH
```

Comment:
Without this, the macos-latest runner doesn’t find the libpq lib and the `pg` gem can’t be compiled properly.